benchmarks/polys.py

```diff
@@ -111,12 +194,24 @@ def setup(self, n, impl):
         if impl == 'expr':
             func = self.get_func_expr(*examples.as_expr())
             expected = examples.to_expr(expected)
+
         elif impl == 'dense':
             func = self.get_func_poly(*examples.as_poly())
-            expected = examples.to_poly(expected)
+            if isinstance(expected, list):
+            # for the subresultants type methods the output is in form of list.
+                expected = [examples.to_poly(polynomial) for polynomial in
+                            expected]
+            else:
+            # for those methods whose output is only a polynomial.
+                expected = examples.to_poly(expected)
```

Comment:
The formatting here is weird. Usually a comment should line up with the line below. Also I would not break a line in a list comprehension. Either allow the line to be a little longer than usual or shorten the code somehow:
```python
            if isinstance(expected, list):
                # some methods output a list of polynomials
                expected = [examples.to_poly(p) for p in expected]
            else:
                # others output only a single polynomial.
                expected = examples.to_poly(expected)
```